src/app/pages/storage/volumes/zvol/zvol-form/zvol-form.component.ts

```diff
@@ -452,7 +452,7 @@ export class ZvolFormComponent implements Formconfiguration{
       _.find(this.fieldConfig, {name:'inherit_encryption'}).placeholder = inherit_encrypt_placeholder;
         
 
-      if (this.isNew) {
+      if (this.isNew && this.legacy_encryption) {
```

Comment:
Can you please clarify if this mean that users can't create encrypted zvols on non-geli pools ?

Reply:
Indeed, this check sounds wrong, @dszidi